RueLaLaTech/django-haystack
# Add Django 1.11 Support

This pull request adds support for Django 1.11.

Main features:
 - the version of Django has been updated to include Django 1.11
 - a tox configuration that now runs tests against Django1.11.

To test:
 - set up this project locally, and run all tests with `tox`.
 - run the `ruetech` repo locally
 - install the `django_1_11` branch of `haystack` in each of the containers that uses it (cobra-storefront, rue-storemanager, and rue-storefront) by changing each apps’s `setup.py` file to install savory-pie from GitHub: change `'django-haystack-rgg-fork==2.5.1’,` to `'django-haystack @ git+https://yourusername:youraccesstoken@github.com/RueLaLa/django-haystack.git@django_1_11'` (fill in your username and GitHub access token) in the following files:
   - src/rue-storemanager/setup.py
   - src/rue-synchronizer/setup.py
 - rebuild the Docker containers
 - make sure that the site still runs as before
 - run all of the tests, and make sure that they all pass, and don’t have any Django1.11 warnings for savory pie
